src/taskgraph/decision.py

```diff
@@ -264,6 +272,38 @@ def _determine_more_accurate_base_ref(repo, candidate_base_ref, head_ref, base_r
     return base_ref
 
 
+def _determine_more_accurate_base_rev(repo, base_ref, candidate_base_rev, head_rev):
+    base_ref_or_rev = candidate_base_rev
```

Comment:
nit: I think the logic would be a bit easier to follow if this goes into an `else:` clause. I was wondering why you do this assignment up here, only to compare using `candidate_base_rev` below, but then realized this was acting as the else condition.

Reply:
Sounds good. `else` condition added!